integration_tests/__tests__/mock_names.test.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Can you add a test for failing `expect(mockWithName).not.toHaveBeenCalled()` and `xpect(mockWithName).toHaveBeenCalledTimes(5)` to assert on the name for them as well? I see no reason for them not to work as you expect, but having the assertions would be nice

Reply:
So, a test where the assertion `expect(mockWithName).not.toHaveBeenCalled()` will fail, and then check for that error?

And a test for `expect(mockWithName).toHaveBeenCalledTimes(5)` when it's not called 5 times?

Reply:
Ok, I've added more tests to cover those.

Reply:
Yup. Thanks!